pages/about.html

```diff
@@ -4,9 +4,28 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="styles/style.css" type="text/css">
+  <link rel="icon" type="image" href="images/favicon.ico">
+  <!-- The navigation menu -->
+  <div class="navbar">
+    <a class="active" href="/index.html">Home</a> 
+    <a href="/about.html">About</a> 
+    <a href="/portfolio.html">Portfolio</a> 
```

Comment:
Since these are in the same project.

```suggestion
    <a href="about.html">About</a> 
    <a href="portfolio.html">Portfolio</a> 
```